modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/AnalyticsEvent.kt

```diff
@@ -471,7 +471,7 @@ enum class AnalyticsEvent(val key: String) {
     SETTINGS_ABOUT_SHARE_WITH_FRIENDS_TAPPED("settings_about_share_with_friends_tapped"),
     SETTINGS_ABOUT_WEBSITE_TAPPED("settings_about_website_tapped"),
     SETTINGS_ABOUT_INSTAGRAM_TAPPED("settings_about_instagram_tapped"),
-    SETTINGS_ABOUT_TWITTER_TAPPED("settings_about_twitter_tapped"),
+    SETTINGS_ABOUT_X_TAPPED("settings_about_twitter_tapped"),
```

Comment:
I have left the analytic name the same so it's easier.

Reply:
I agree with this. I can also keep the tap data history